### DISCUSSION TITLE
`withoutDefaultToolbarButtons()` for `RichText`

### DISCUSSION THREAD

**Discussion by ryangjchandler** [filamentphp/filament#36] [Ideas]
The documentation is a bit misleading and it feels like calling `enableToolbarButtons` should **only** enable the buttons specified in the array argument.

The way to fix this would be disabling all the buttons you don't want using `disableToolbarButtons` or disabling all buttons and only enabling the ones you do want.

I'm proposing a new `withoutDefaultToolbarButtons` (or similarly named) method that just empties out `$toolbarButtons`.

**Comment by danharrin** [comment]
Implemented in v1.0.7 as `disableAllToolbarButtons()`.
